NitroxServer-Subnautica/Resources/Parsers/PrefabPlaceholderGroupsParser.cs

```diff
@@ -48,17 +48,9 @@ public PrefabPlaceholderGroupsParser()
 
         // Select only prefabs with a PrefabPlaceholdersGroups component in the root ans link them with their dependencyPaths
         ConcurrentDictionary<string, string[]> prefabPlaceholdersGroupPaths = GetAllPrefabPlaceholdersGroupsFast(loadAddressableCatalog);
-        am.UnloadAll();
```

Comment:
This line should stay bc the internal cache list is slowing down the process more than loading a few assets again. This is maybe better in new AssetToolsNetVersion but we need a byte to texture library bc ATNs sub-package is only for netStadnard.

Reply:
This would have been great as comment in code. Will add it back later. On my PC server starts in 3.6s so didn't notice.